components/src/dynamo/profiler/tests/unit/test_rapid_backend_consistency.py

```
@@ -0,0 +1,242 @@
+        model=_MODEL,
+        backend=backend,
+        image=image,
+        hardware=HardwareSpec(gpuSku="h200_sxm", totalGpus=8, numGpusPerNode=8),
+        workload=WorkloadSpec(isl=4000, osl=1000),
+        sla=SLASpec(ttft=2000.0, itl=50.0),
+    )
+
+
+def _make_task(backend: str, serving_mode: str = "agg") -> Task:
+    """A real AIC Task for one backend, matching `_make_dgdr()`."""
+    shared = dict(total_gpus=8, isl=4000, osl=1000, ttft=2000.0, tpot=50.0)
```

Ruff (0.16.3)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)